### DISCUSSION TITLE
Galaxy Brain blabla Q1

### DISCUSSION THREAD

**Discussion by lueluew** [Schweinepriester/github-profile-achievements#338] [Q&A] [answered]
Galaxy Brain blabla Q1

**Comment by lueluew** [comment]
11

**Comment by cccy0** [comment, marked as answer]
2 accepted answers ~ 🧢

**Comment by WHAHA-HA** [comment]
Actually, I can achieve Quickdraw and Pull Shark badges with ease, but no Galaxy Brain.
PS: It'd be appreciated if you could mark my answer
